test/mocks/urlfetchapp.js

```diff
@@ -0,0 +1,28 @@
+var Future = require('fibers/future');
+
+var MockUrlFetchApp = function() {
+  this.delayFunction = () => 0;
+  this.resultFunction = () => '';
+};
+
+MockUrlFetchApp.prototype.fetch = function(url, optOptions) {
```

Comment:
- Please document these methods. I don't know what these methods (giving 0 and '') do.
- The params don't seem to be used.

Reply:
They are mocking out the equivalent functions in UrlFetchApp, added a note. I decided to keep the same method signature, even though this implementation currently isn't use those parameters.